tests/test_git_source.py

```diff
@@ -248,3 +248,12 @@ def test_enumerate_versions_with_functionally_duplicate_sources():
         )
         assert 1 == ev.call_count
         assert [[v1, v2]] == versions
+
+
+def test_strip_pr_number():
+    strip = autosynth.git_source._strip_pr_number
+    assert "fix bugs", strip("fix bugs (#244)")
```

Comment:
these assertions look incorrect
```suggestion
    assert "fix bugs" == strip("fix bugs (#244)")
```

Reply:
Doh.  Thanks!